tools/cassandra/handler.go

```diff
@@ -207,10 +200,7 @@ func validateCQLClientConfig(config *CQLClientConfig, isDryRun bool) error {
 		return schema.NewConfigError("missing cassandra endpoint argument " + flag(schema.CLIOptEndpoint))
 	}
 	if config.Keyspace == "" {
-		if !isDryRun {
```

Comment:
nit: looks like `isDryrun` is not used in the function any more. Can we remove it from the parameter?

Reply:
Good catch. Removed.